release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1772,6 +1772,18 @@ The issue might still occur with some combinations of input for isolated CPUs in
 [id="ocp-release-note-storage-bug-fixes_{context}"]
 ==== Storage
 
+* Previously, Portworx plugin Container Storage Interface (CSI) migration fails without inclusion of an upstream patch. With this release, the Portworx plugin CSI translation now copies the secret name and namspace to Kubernetes version to 1.31 so that an upstream patch is not required. (link:https://issues.redhat.com/browse/OCPBUGS-49437[*OCPBUGS-49437*])
+
+* Previously, the VSphere Problem Detector Operator waited up to 24 hours to reflect changing `clustercsidrivers.managementState` from `Managed` to `Removed` for a {vmw-full} cluster. With this release, the VSphere Problem Detector Operator now relfects such a state change to about 1 hour. (link:https://issues.redhat.com/browse/OCPBUGS-39358[*OCPBUGS-39358*])
+
+* Previously, the Azure File Driver attempted to reuse existing storage accounts. With this release, the Azure File Driver creates storage accounts during dynamic provisioning. For updated clusters, newly-created Persistent Volumes use a new storage account. Persistent Volumes that were previously provisioned continue using the same storage account used before the cluster update. (link:https://issues.redhat.com/browse/OCPBUGS-38922[*OCPBUGS-38922*])
+
+* Previously, the configuration loader logged `yaml` unmarshall errors when the INI succeeded. With this release, the unmarshall errors are no longer logged when the INI succeeds. (link:https://issues.redhat.com/browse/OCPBUGS-38368[*OCPBUGS-38368*])
+
+* Previously, the Storage Operator counted an incorrect amount of control plane nodes that existed in a cluster. This count is needed for the Operator to determine the number of replicas for controllers. With this release, a fix applied to the Storage Operator means that the Operator now counts the correct amount control plane nodes that in turn lead to a more accurate count of replica controllers. (link:https://issues.redhat.com/browse/OCPBUGS-36233[*OCPBUGS-36233*])
+
+* Previously, the `manila-csi-driver` and node registrar pods had missing healthchecks because of a configuration issue. With this release, the health checks are now added to both of these resources. (link:https://issues.redhat.com/browse/OCPBUGS-29240[*OCPBUGS-29240*])
```

Comment:
```suggestion
* Previously, the `manila-csi-driver` and node registrar pods had missing health checks because of a configuration issue. With this release, the health checks are now added to both of these resources. (link:https://issues.redhat.com/browse/OCPBUGS-29240[*OCPBUGS-29240*])
```